tests/Basic/Basic.h

```diff
@@ -760,15 +760,10 @@ class TestNamingAnonymousTypesInUnion
 };
 
 #define CS_API
-class CS_API ClassPassTry
-{
-public:
-	int n;
-	char c;
-};
+class CS_API RefTypeClassPassTry { };
 
-void funcTry(CS_OUT ClassPassTry* classTry) { }
-void funcTry2(CS_OUT ClassPassTry classTry) { }
+void DLL_API funcTryRefTypePtrOut(CS_OUT RefTypeClassPassTry* classTry) { }
```

Comment:
As usual, could you please replace these inlined implementations with real ones (in the CPP)?